packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -2522,3 +2606,206 @@ class _RenderLayoutSurrogateProxyBox extends RenderProxyBox {
     }
   }
 }
+
+class _OverlayChildLayoutBuilder extends AbstractLayoutBuilder<(Size, Matrix4, Size)> {
+  const _OverlayChildLayoutBuilder({required this.userSuppliedBuilder});
+
+  final OverlayChildLayoutBuilder userSuppliedBuilder;
+
+  Widget _builder(BuildContext context, (Size, Matrix4, Size) layoutInfo) {
+    return userSuppliedBuilder(context, layoutInfo.$1, layoutInfo.$2, layoutInfo.$3);
```

Comment:
Even more indication that we maybe should just define an explicit immutable class for the parameters to the builder. It could be re-used here and elsewhere in this class. The tripplet seems to add a lot of noise that make this class look more complicated than it really is.